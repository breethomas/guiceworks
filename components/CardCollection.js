// @flow
import React from 'react'
import css, { media2 } from '../styles/css'
import View from './View'

type Props = {
  children: React.Element<*>
}

const style = css(
  {
    display: 'flex',
    flexFlow: 'column wrap',
    maxWidth: 1400,
    marginRight: 'auto',
    marginLeft: 'auto',
  },
  media2({ flexFlow: 'row wrap' }),
)

export default ({ children, ...props }: Props) => (
  <View className={style} {...props} >
    { children }
  </View>
)
