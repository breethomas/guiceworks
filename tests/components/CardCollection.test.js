import React from 'react'
import renderer from 'react-test-renderer'
import CardCollection from '../../components/CardCollection'

describe('CardCollection', () => {
  let component

  beforeEach(() => {
    component = (
      <CardCollection>
        <span>CardCollection</span>
      </CardCollection>
    )
  })

  it('shows the correct snapshot for the CardCollection component tree', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
