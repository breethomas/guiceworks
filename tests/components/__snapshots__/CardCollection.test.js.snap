// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardCollection shows the correct snapshot for the CardCollection component tree 1`] = `
<div
  className={
    Object {
      "data-css-nz4u6x": "",
    }
  }
>
  <span>
    CardCollection
  </span>
</div>
`;
